Pod/Classes/VOKMappableModel.h

```diff
@@ -0,0 +1,32 @@
+//
+//  VOKMappableModel.h
+//  VOKCoreData
+//
+
+#import "VOKCoreDataManager.h"
+
+/**
+ *  Any models that conform to this protocol will be automatically registered for mapping with the shared instance 
```

Comment:
Actually, the way I've currently implemented this, they're automatically registered for mapping with *any* instance of `VOKCoreDataManager`.  Anyone have thoughts on that?  Is it a problem to do the automatic registration on init for any instance?  To do automatic registration without any way to turn it off?